packages/Icon/src/Icon.spec.tsx

```diff
@@ -6,12 +6,20 @@ afterAll(cleanup);
 
 const text = 'Hello world';
 
+// eslint-disable-next-line react/display-name
+const MyGlyph: React.FC<React.SVGProps<SVGSVGElement>> = () => (
```

Comment:
Testing this component was interesting...

I don't know if this will cause problems, but maybe the fake glyph should render an SVG if we're typing it as returning such...

Reply:
Technically I'm pretty sure that all it's claiming here is that it can accept the same props as an SVG element, and that if any event handlers are called `event.target` will be an SVG. None of that is inaccurate, exactly...

Reply:
Fair enough!